src/libclang.nim

```diff
@@ -1,7 +1,13 @@
 from times import Time
 {.deadCodeElim: on.}
 {.push callconv: cdecl.}
-when defined(windows): 
+
+when defined(overrideLibclangName):
+  # Provide a way to let the user specify the library name from the outside
+  const
+    overrideLibclangName {.strdefine.} = ""
+    libclang* = overrideLibclangName
```

Comment:
Wouldn't it be better to just use `dynlibOverride` as a common way for such things?

Reply:
Maybe, but then I'd be forced to statically link against libclang...